openedx/core/djangoapps/agreements/api.py

```diff
@@ -68,3 +70,76 @@ def get_integrity_signatures_for_course(course_id):
     """
     course_key = CourseKey.from_string(course_id)
     return IntegritySignature.objects.filter(course_key=course_key)
+
+
+def get_ltipiitool(course_id):
```

Comment:
I'd like to see a function name that is less tied to the underlying model name. Maybe something like `get_pii_receiving_lti_tools`? Or whatever you think is best. I know it's virtually the same, but the current functions name reads like it's returning a model instance - because it is - and I don't think it should return a model instance - please see my overall pull request comment.